mRemoteV1/Resources/Help/ui_menus.htm

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <meta charset="utf-8">
+  <title>Menus</title>
+  <link rel="stylesheet" href="Main.css" type="text/css">
+</head>
+<body>
+  <div class="isa_error">
+    In progress - needs review and help to make it correct with information
+  </div>
+
+  <!-- Introduction - #start -->
+  <h1>Introduction</h1>
+  <hr>
+  <img src="Screenshots/Main Menu/Main_Menu.png">
+  <p>
+    In this section we are going to explain the menus located in mRemoteNG. The above screenshot shows the main menu with colors. Short color explanation:
+    <p style="margin-left: 40px;">
+      <span style="color: crimson;">Red</span> - Anchor to move menu around the interface<br>
+      <span style="color: forestgreen;">Green</span> - The menu items
+    </p>
+  </p>
+  <h2>Quick Reference</h2>
+  <ul>
+    <li><a href="#file_menu"><b>File Menu</b></a> - Contains standard commands for the application</li>
+    <li><a href="#view_menu"><b>View Menu</b></a> - Menu for additional dialogs for mRemoteNG</li>
+    <li><a href="#tools_menu"><b>Tools Menu</b></a> - Additional tools that can be used and triggered in mRemoteNG</li>
+    <li><a href="#help_menu"><b>Help Menu</b></a> - Get more information for the application</li>
+  </ul>
+  <!-- Introduction - #end -->
+
+  <!-- File menu - #start -->
+  <h1 id="file_menu">File Menu</h1>
+  <hr>
+  <table>
+    <tr>
+      <th>Item</th>
+      <th>Description</th>
+    </tr>
+    <tr>
+      <td>New Connection</td>
+      <td>Will add a new connection to the Connections dialog after where the cursor is positioned.</td>
+    </tr>
+    <tr>
+      <td>New Folder</td>
+      <td>Add a new folder in the Connections dialog tree where the cursor is positioned.</td>
+    </tr>
+    <tr>
+      <td>New Connection File</td>
+      <td>Create a new connection file. Dialog will come up ask about: filename and where to place the new connection file.</td>
+    </tr>
+    <tr>
+      <td>Open Connection File</td>
+      <td>
+        Open a connection file. Dialog comes up asking about which file to open. For security reasons, this also shows a dialog
+        to ask if you want to save the current file before continuing.
+      </td>
+    </tr>
+    <tr>
+      <td>Save Connection File</td>
+      <td>Saves the current connection file open. If you are using a SQL server connection instead it will send a save to the SQL server.</td>
+    </tr>
+    <tr>
+      <td>Save Connection File As...</td>
+      <td>Saves the current connection file to a specific location on disk.</td>
+    </tr>
+    <tr>
+      <td>Delete...</td>
+      <td>Delete current selected item in connections dialog.</td>
```

Comment:
For this and the next 3 items, change "current selected" to "currently selected"